native/ios/swift/MeetHourSDKTest/Podfile

```diff
@@ -1,7 +1,7 @@
 platform :ios, '12.1'
 
 target 'MeetHourSDKTest' do
-    pod 'MeetHourSDK', '~> 3.5.2'
+    pod 'MeetHourSDK', '~> 3.5.0'
```

Comment:
Latest version is 3.5.2. @chepury-bharathraj

Reply:
Done.